pgjdbc/src/main/java/org/postgresql/jdbc/PgStructField.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2004, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.jdbc;
+
+public class PgStructField {
```

Comment:
final

Reply:
Renamed this to PgAttribute as it seems more appropriate